purescript.cabal

```diff
@@ -119,7 +119,7 @@ common defaults
     TypeFamilies
     ViewPatterns
   build-tool-depends:
-    happy:happy ==1.20.0
+    happy:happy ^>= 1.20.0
```

Comment:
haskell.nix couldn't understand it and it was throwing later at build time on version missmatch